app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    _, old_name, new_name = command.split()
+    try:
+        os.renames(old_name, new_name)
```

Comment:
Code Efficiency: Using `os.renames` can lead to errors if the directory already exists. Consider checking if the directory exists before attempting to rename or move the file.